frontend/src/app/problems/add/execution-constraints.tsx

```
@@ -0,0 +1,67 @@
+                                        type="number"
+                                        fullWidth
+                                        id="time-limit"
+                                        label="Time Limit"
+                                        variant="outlined"
+                                        placeholder="Time Limit"
+                                    />
+                                </TableCell>
+                                <TableCell>
+                                    <TextField
+                                        value={memoryLimit}
+                                        onChange={(e) => setMemoryLimit(parseInt(e.target.value))}
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)